### DISCUSSION TITLE
Purge old event data?

### DISCUSSION THREAD

**Discussion by der-On** [plausible/analytics#1436] [Self-Hosted Support] [answered]
I'm self hosting plausible and my `event/store` folder grew in size pretty fast and is now at 37GB. I've recently increased the disk space but it filled up very fast again. As I do not need all the data, I would like to be able to purge old events to regain disk space. Is there any command already available to do this or if not is there something planned? Or is this rapid increase not as intended?

**Comment by der-On** [comment, marked as answer]
Seem's like clickhouse logging and got resolved here https://github.com/plausible/analytics/discussions/588

**Reply by der-On** [reply]
Yep, was able to resolve the problem by purging clickhouse log data as suggested in #588 and with the updated clickhouse config that disables logging. Got my `event/store` folder down to some megabytes.

**Reply by ukutaht** [reply]
Great! Indeed, Clickhouse is very eager with logging stuff and that would have been my first suggestion :)

**Comment by hiepxanh** [comment]
I turn off logging and still not solve the problem. after 1 year, my disk grow to 30gb. is that normal to you @ukutaht ?
